docs/using-radicle/creating-and-sharing-projects.md

```diff
@@ -17,9 +17,9 @@ At the moment, [projects can only be created through the Upstream client](unders
 
 ## Sharing projects
 
-Projects can be shared with their projectID. The projectID can be found under the project name on the project page.
+Projects can be shared with their Radicle. The Radicle ID can be copied from the project page.
```

Comment:
```suggestion
Projects can be shared with their Radicle ID. The Radicle ID can be copied from the project page.
```

Reply:
Why the ID? It's the URN that is shared.

Reply:
@xla `URN` -> `Radicle ID` as outlined here: https://github.com/radicle-dev/radicle-upstream/pull/1206#issuecomment-724317000.

Reply:
:exploding_head: